src/components/Banner/Banner.tsx

```diff
@@ -52,15 +53,45 @@ const ContentWrapper = styled(Padbox)`
 
 /* stylelint-disable */
 const Text = styled(BaseText)<{ $variant?: BannerProps['variant'] }>`
-  max-width: 125ch;
+  max-width: 150ch;
```

Comment:
Be careful, `ch` unit can be tricky and not always corresponds with the real number of characters for given font. E.g. for Inter it should be `115ch` to get 150 characters per line. So let's change it to `115`
https://meyerweb.com/eric/thoughts/2018/06/28/what-is-the-css-ch-unit/